api_spec_oa3.yml

```diff
@@ -58,34 +97,43 @@ paths:
             validate the existence of asset files like logos and
             screenshots.
       responses:
-        200:
-          description: |-
-            Validation Ok, return latest valid publiccode version
+        '200':
+          description: 'Validation Ok, return latest valid publiccode version'
```

Comment:
@sebbalex why don't `|-`? It makes code more readable. `description` is supposed to be long enough to document the response.